AUTHORS

```diff
@@ -1296,20 +1296,82 @@ Abhishek kumar <kumar325571@gmail.com>
 Sam Lubelsky <sammy56lt@gmail.com>
 Henrique Soares <henrique.c.soares@tecnico.ulisboa.pt>
 Vladimir Sereda <voffch@gmail.com>
+Hwayeon Kang <hwayeonniii@gmail.com>
 Raj Sapale <raj4sapale4@gmail.com>
 Gerald Teschl <gerald.teschl@univie.ac.at>
 Richard Samuel <98638849+samuelard7@users.noreply.github.com>
 HeeJae Chang <hechang@microsoft.com>
-Hwayeon Kang <hwayeonniii@gmail.com>
 Nick Harder <nharder@umich.edu>
 Ethan DeGuire <ethandeguire@gmail.com>
 Lorenz Winkler <lorenz.winkler@tuwien.ac.at>
 Richard Rodenbusch <rrodenbusch@gmail.com>
 Zhenxu Zhu <xzdlj@outlook.com>
 Mark van Gelder <m.j.vangelder@student.tudelft.nl>
+Mark van Gelder <mvgmvgmvg@live.com>
+Ishan Pandhare <91841626+Ishanned@users.noreply.github.com>
 James A. Preiss <jamesalanpreiss@gmail.com>
 Emile Fourcini <emile.fourcin1@gmail.com>
 Alberto Jiménez Ruiz <Alberto.Jimenez@uclm.es>
 João Bravo <joaocgbravo@tecnico.ulisboa.pt>
 Dean Price <dean1357price1357@gmail.com>
+Edward Z. Yang <ezyang@mit.edu>
+James Titus <titusjames299@gmail.com>
+Zhuoyuan Li <zy.li@stu.pku.edu.cn>
 Hugo Kerstens <hugo@kerstens.me>
+Jan Jancar <johny@neuromancer.sk>
+Andrew Mosson <amosson@yahoo.com>
+Marek Madejski <marekmadejski@yandex.com>
+Gonzalo Tornaría <tornaria@cmat.edu.uy>
+Peter Stahlecker <peter.stahlecker@gmail.com>
+Jean-François B <2589111+jfbu@users.noreply.github.com>
+Zexuan Zhou (Bruce) <zzx498636727@gmail.com>
+George Frolov <gosha@fro.lv>
+Corbet Elkins <corbet286@gmail.com>
+Håkon Kvernmoen <haakon.kvernmoen@gmail.com>
+Muhammad Maaz <mmaaz6004@gmail.com>
+Shishir Kushwaha <138311586+shishir-11@users.noreply.github.com>
+Matt Wang <mattwang44@gmail.com>
+bharatAmeria <21001019007@jcboseust.ac.in>
+Amir Ebrahimi <github@aebrahimi.com>
+Steven Esquea <steven.esquea@irreverente.net>
+Rishabh Kamboj <111004091+VectorNd@users.noreply.github.com>
+Aasim Ali <aasim250205@gmail.com>
+Ivan A. Melnikov <iv@altlinux.org>
+Borek Saheli <boreksaheli@gmail.com>
+Guido Roncarolo <guido.roncarolo@gmail.com>
+Quek Zi Yao <ziyaoqzy2001@gmail.com>
+Roelof Rietbroek <r.rietbroek@utwente.nl>
+MostafaGalal1 <mostafag649.mg@gmail.com>
```

Comment:
@MostafaGalal1 is this how you want your name to be recorded in the AUTHORS file?